.github/workflows/ci.yaml

```diff
@@ -31,7 +31,7 @@ concurrency:
   cancel-in-progress: true
 
 jobs:
-  build:
+  compile-on-jdk8:
     runs-on: self-hosted
```

Comment:
The compile can run on github `ubuntu-latest` to reduce resource cost on our own hosted runners.

Reply:
We don't need run compile for different modules for compiling java8.

Reply:
Add a `name: "Compile Java 8"`